client/src/pages/Tree/TreeEdit.js

```diff
@@ -95,15 +95,12 @@ export default function TreeEdit({
     onConfirm(formData);
   };
 
-  const handleError = (errorLog, e) => console.error('errors, e', errorLog, e);
-
   return (
     <FormScrollableDialog
       title="Edit Tree"
       open={open}
       onConfirm={handleConfirm}
       onCancel={onCancel}
-      onError={handleError}
```

Comment:
I don't think these changes should be part of this PR. These would be part of #342.